controllers/openstackplaybookgenerator_controller.go

```diff
@@ -240,7 +240,7 @@ func (r *OpenStackPlaybookGeneratorReconciler) Reconcile(ctx context.Context, re
 			if common.StringInSlice(roleParams.RoleName, fencingRoles) && roleParams.RoleCount == 3 {
 				// Get the associated VM instances
 				virtualMachineInstanceList, err := common.GetVirtualMachineInstances(r, instance.Namespace, map[string]string{
-					common.OwnerNameLabelSelector: roleName,
+					common.OwnerNameLabelSelector: strings.ToLower(roleName),
```

Comment:
I think the OwnerNameLabelSelector was always the object.Name [1] which afaik can only be lower case.
If we want the non lower ooo role name, we could add a label in [2] and use that.

[1] https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/pkg/common/labels.go#L27
[2] https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/controllers/openstackvmset_controller.go#L1068

Reply:
You're right.  In this case, though, the `OsVMSet` name is dictated by a RoleName from the `OsControlPlane`.  So, in turn, the `VirtualMachineInstance`s owner labels (the children of the `OsVMSet`) ultimately have the lowercase role name for `osp-director.openstack.org/name`.